panel/io/compile.py

```diff
@@ -63,15 +63,18 @@ def setup_build_dir(build_dir: str | os.PathLike | None = None):
             shutil.rmtree(temp_dir)
 
 
-def check_cli_tool(tool_name):
+def check_cli_tool(tool_name: str) -> bool:
```

Comment:
Maybe just use `shutil.which` to detect instead of launching a subprocess.

Reply:
Done, thanks!